LemonWay/Models/KycDoc.php

```diff
@@ -76,4 +76,4 @@ function __construct($node) {
 	}
 }
 
-?>
\ No newline at end of file
+?>
```

Comment:
Yes, but the Github editor automatically added a line feed at the end of the file (this is a good practice, when there is no closing tag)

![image](https://cloud.githubusercontent.com/assets/1501825/24092730/6147cde8-0d50-11e7-8012-48ac53fec814.png)

Anyway, I am not part of Lemonway team :)